ownerdelete.user.js

```diff
@@ -217,10 +227,13 @@ dry.once("load", () => {
         {white: true} :
         {green: true});
       file.dom.setTags(tags);
+      getInfo(file.id).then(result => {
```

Comment:
Lain will hate you a) for getting the info ALWAYS and b) for getting all infos at once basically. (A 1000 file room would dispatch 1000 getinfo requests at once).

You should probably a) add a cache<sup>1</sup>, that is somewhat persistent but not too persistent (e.g. sessionStorage) and b) use something (like promisepool as volanail does) to limit the number of concurrent in-flight requests.

<sup>1</sup> something I considered to do for volanail but it's a little harder there as the information nail uses may be updated later, while the checksum will not update so that's not an issue here